src/controller.cpp

```diff
@@ -42,23 +42,48 @@ namespace bzzz
         int &motorFR,
         int &motorBL,
         int &motorBR,
+        int &motorML,  // adding for new motor middle left
```

Comment:
`motorML` and `motorMR` are not (or, at least, should not be) defined for quadcopters